scripts/validate_json.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
This file should be renamed to match the naming convention of the python check scripts, so `check_json_formatting.sh`.

Reply:
Done.